backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -1058,7 +1059,7 @@ def make_path_object(self, data: typing.Dict[str, typing.Any]) -> object:
 
 
 class KnownMembersQuerySchema(marshmallow.Schema):
-    acp = StrippedString(example="test", description="search text to query", required=True)
+    acp = StrippedString(example="test", description="search text to query", required=False)
```

Comment:
We need to log this type of change somewhere to not forget it during the writing the changelog

I'll check how we use to do so

Reply:
documented in https://github.com/tracim/tracim/issues/6280